stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/component_browser/messages/viewholder/PlainTextWithFileAttachmentsMessagesComponentBrowserFragment.kt

```diff
@@ -144,37 +143,37 @@ class PlainTextWithFileAttachmentsMessagesComponentBrowserFragment : BaseMessage
 
         val attachmentPdf = Attachment(
             type = "file",
-            mimeType = ModelType.attach_mime_pdf,
+            mimeType = "application/pdf",
```

Comment:
Can't we use constants for those?

Reply:
Hm, I moved mime type constants to the internal UI module. It it not exposed to the sample app. Any suggestions?

Reply:
Ohh, I see, this is the sample app....
If we are using it there, should it be needed by our customers as well???
What about exposing those constants as public constants? :thinking:

Reply:
If we decide to expose mime types we should carefully review all of them so we don't expose invalid ones like